sim/wait-for-it-quic/wait-for-it.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"bytes"
+	"flag"
+	"fmt"
+	"log"
+	"net"
+	"time"
+	"os"
+)
+
+// compose a packet that will elicit a Version Negotiation packet
+var packet = append([]byte{0xc0, 0x57, 0x41, 0x49, 0x54, 0, 0}, make([]byte, 1200)...)
```

Comment:
Cute :-)

Reply:
Actually, this uncovered a problems with VN handling at a bunch of servers: https://github.com/quicwg/base-drafts/issues/4186.

Reply:
I meant Version = "WAIT" is cute